### DISCUSSION TITLE
Twitch stream not working on SmPlayer.

### DISCUSSION THREAD

**Discussion by c2b88** [smplayer-dev/smplayer#719] [General]
Since yesterday Twitch streams are no longer working on SmPlayer. Was able to drag and drop Twitch live streams the same way like Youtube, but since yesterday I am no longer able to do so. 
Would appreciate it if someone is able to resolve this issue. Thanks.

**Comment by knglerxst** [comment]
Same here.

**Comment by c2b88** [comment]
Its started working now...so closing this.
